core-web/libs/portlets/dot-content-drive/portlet/src/lib/dot-content-drive-shell/dot-content-drive-shell.component.scss

```diff
@@ -35,11 +35,13 @@ dot-content-drive-toolbar {
     border-right: 1px solid $color-palette-gray-400;
     width: 0;
 
-    transition: width 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    visibility: hidden;
 
     &.expanded {
         /* This is the width according to figma */
         width: 16.5rem;
+        visibility: visible;
```

Comment:
What this do here?

Reply:
this is to hide/show the tree selector, I added it on my open [PR](https://github.com/dotCMS/core/pull/33013) in order to fix an issue we have on the table, we needed to merge both branches to test some things out, maybe kev committed it by error